.github/workflows/build.yml

```diff
@@ -87,11 +87,13 @@ jobs:
 
   check_abi:
     name: 'Check if the ABI has changed'
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04  # 24.04 causes spurious errors
```

Comment:
Do you know if these are the same errors that we just saw about the missing bits?